backend/python/app/services/implementations/user_service.py

```diff
@@ -150,6 +150,20 @@ def get_users(self, isTranslators, language=None, level=None, name_or_email=None
                 )
                 raise e
 
+        if (level is not None) and (not language or language == ""):
```

Comment:
nit: `if level and not language` should be sufficient, `""` is falsey

Reply:
ah the conditional looks like that because I took the negation of this statement: 
![image](https://user-images.githubusercontent.com/54864799/139605944-8aec4f90-da61-4d2b-a2b1-7cf957915a88.png)